### PR TITLE
Add DocuSeal to Document Management

### PR DESCRIPTION
**Pull request by kokomo123**
I think docuseal is now mature and I know it's ready for the awesome-selfhosted list as it's past the 4 month mark at this point. Decided to add it myself 👍🏻 As usual, if there's any comments or concerns please let me know and i'll be sure to adjust this PR If necessary. 

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [X] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [X] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [X] Comments and unused optional fields have been removed.
- [X] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [X] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [ ] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [X] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/docuseal.yml
Comment (nodiscc) [inline software/docuseal.yml:10]:
```suggestion
  - Docker
```

I can't find documentation on how to install from source, Docker seems to be the only supported installation method.

File: software/docuseal.yml
Comment (nodiscc) [inline software/docuseal.yml:13]:
```suggestion
```

The default value is already `no`, lets remove this

**Review by nodiscc — CHANGES_REQUESTED** [review]
> it's past the 4 month mark

Not yet (https://github.com/docusealco/docuseal/releases?page=3), first releases were made in July 2023, so let's wait until 2023/11/10 before we merge this.

Thanks!

**Comment by kokomo123** [comment]
Yeah... i was including july-october. rip

**Comment by Ki-er** [comment]
Date is 23/11/10 so this is now mature enough!

**Comment by kokomo123** [comment]
Woohoo!

**Review by nodiscc — APPROVED** [review]
Thanks

**Comment by kokomo123** [comment]
We've certainly sealed the deal on this one (had to wait to make that joke but it was worth it)

**Pull request merged**
